tests/models/test_dag.py

```diff
@@ -94,10 +95,17 @@
 from tests.test_utils.timetables import cron_timetable, delta_timetable
 
 TEST_DATE = datetime_tz(2015, 1, 2, 0, 0)
-
 repo_root = Path(__file__).parents[2]
 
 
+def convert_dt(tz: Timezone, dt, dst_rule):
+    """Converts a datetime in the current timezone"""
+    try:
+        return tz.convert(dt=dt, dst_rule=dst_rule)
+    except TypeError:  # Pendulum 3 Timezone doesn't have dst_rule
+        return tz.convert(dt=dt)
```

Comment:
Hmm this is awkward. How does Pendulum 3 handle time being rolled back? (`fold` in Python timezone speak.) Simply dropping the argument seems wrong, although this is probably OK-ish for just tests.

Reply:
If I do not miss anything then `dst_rule` used for compatibility with Python < 3.6
https://pendulum.eustace.io/docs/#using-the-timezone-library-directly

I think it use `fold` argument from datetime.datetime in current implementation: https://github.com/sdispater/pendulum/blob/18f9c0ec365cb5ddf5f90722535a46ecfea7bd9b/src/pendulum/tz/timezone.py#L107-L114

I could change a bit test helper

Reply:
Maybe we should modify the tests instead to remove `dst_rul` entirely. Airflow doesn’t support 3.6 anymore.

Reply:
> Airflow doesn’t support 3.6 anymore

FYI, Python 3.6 did introduce fold, so this is only for Python 3.5 and earlier.